dev/integration_tests/abstract_method_smoke_test/android/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -2,4 +2,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.7-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.4-all.zip
```

Comment:
To confirm, you update these integration tests by letting the migrator actually run on them, not by editing it manually?

Reply:
I did not - I made these changes before writing the migrator to see if it would cause any errors. I tested on some local projects, but it would be a good sanity check to let the migrator make these changes. I'll revert them locally and let the migrator handle and update with results.

Reply:
It won't get everything, but a bunch of them will be covered/migrated by running the build tests shard locally:
```
$ SHARD=build_tests dart dev/bots/test.dart
```
To save time first comment out all the ` _flutterBuild*` lines in `_runExampleProjectBuildTests` except one of the `_flutterBuildApk` lines:
https://github.com/flutter/flutter/blob/303d0d3e2b7131ed6177ec127f05a46f4307b519/dev/bots/test.dart#L580
That way it doesn't build all the iOS, Linux, etc projects when you only need the Android ones.

Reply:
Thanks, I was trying to run them 1 by 1. I downgraded back to their original versions and ran the tests using that command, and all of the tests that ran then returned to 7.4 as expected. I also tested it out with a couple of differently formatted versions (i.e. major, minor, patch -> major, minor, etc) because these cases were all exactly 6.7 to 7.4.

I'm still working on running the ones that weren't hit by this command

Reply:
I can now confirm they all upgrade to what was already committed.